e2e/scripts/interactive-multi-node-install.exp

```diff
@@ -21,4 +21,8 @@ expect -re "Add another node?.*:" { send "n\r" }
 set timeout 600
 expect "Enter a new Admin Console password:" { send "password\r" }
 expect "Confirm password:" { send "password\r" }
-expect "Admin Console is ready!"
+expect "Confirm password:" { send "password\r" }
```

Comment:
should we really have to type the password three times?

Reply:
My mistake. Removed.